### PR TITLE
Inject uuid in images in addition to config

### PR DESCRIPTION
**Pull request by kasperl**

### CODE REVIEW

File: tools/inject_config.toit [hunk 1]
```diff
@@ -32,31 +32,37 @@ IMAGE_DATA_OFFSET ::= 296
 IMAGE_DATA_MAGIC_1 := 0x7017da7a
```
Comment (floitsch) [inline tools/inject_config.toit:32]: Should these be constants? (`::=`)

File: tools/inject_config.toit [hunk 2]
```diff
@@ -32,31 +32,37 @@ IMAGE_DATA_OFFSET ::= 296
...
-  result := inject_config config bin_data
+  result := inject_config config unique_id bin_data
 
   out_stream := file.Stream.for_write out_path
   out_writer := writer.Writer out_stream
   out_writer.write result
   out_stream.close
 
-// the factory image contains a "empty" section of 1024 bytes here we encoded the config such that
+// The factory image contains an "empty" section of 1024 bytes here we encoded the config such that
```
Comment (floitsch) [inline tools/inject_config.toit:62]: Where?

**Review by floitsch — APPROVED** [review]
LGTM.

File: tools/inject_config.toit (hunk 2 above)
Comment (kasperl) [reply tools/inject_config.toit]: Done.

File: tools/inject_config.toit (hunk 1 above)
Comment (kasperl) [reply tools/inject_config.toit]: Done.

**Pull request merged**